src/board/v3/error.c

```diff
@@ -30,6 +30,16 @@
  * System low-level error handling and reporting
  * \{ */
 
+#define FAULT_HANDLER_SCREAMING_DEATH(msg_str)                                \
+do {                                                                          \
+  uint32_t *lr;                                                               \
+  asm("mov %0, lr" : "=r"(lr) : : );                                          \
+  char msg[128];                                                              \
```

Comment:
These fault handlers have super small stacks.  We should probably switch to another CPU mode before allocating a big buffer or calling sprintf.